tools/monorepo-utils/src/code-freeze/commands/version-bump/lib/update.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import { readFile, writeFile, stat } from 'fs/promises';
+import { join } from 'path';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../../core/logger';
+
+/**
+ * Update plugin readme changelog.
+ *
+ * @param  tmpRepoPath cloned repo path
+ * @param  nextVersion version to bump to
+ */
+export const updateReadmeChangelog = async (
+	tmpRepoPath: string,
+	nextVersion: string
+): Promise< void > => {
+	const filePath = join( tmpRepoPath, `plugins/woocommerce/readme.txt` );
+	try {
+		const readmeContents = await readFile( filePath, 'utf8' );
+
+		const updatedReadmeContents = readmeContents.replace(
+			/= \d+\.\d+\.\d+ \d\d\d\d-XX-XX =\n/m,
+			`= ${ nextVersion } ${ new Date().getFullYear() }-XX-XX =\n`
+		);
+
+		await writeFile( filePath, updatedReadmeContents );
+	} catch ( e ) {
+		Logger.error( 'Unable to update readme changelog' );
+	}
+};
+
+/**
+ * Update plugin class file.
+ *
+ * @param  tmpRepoPath cloned repo path
+ * @param  nextVersion version to bump to
+ */
+export const updateClassPluginFile = async (
+	tmpRepoPath: string,
+	nextVersion: string
+): Promise< void > => {
+	const filePath = join(
+		tmpRepoPath,
+		`plugins/woocommerce/includes/class-woocommerce.php`
+	);
+
+	try {
+		await stat( filePath );
```

Comment:
Nitpick but `fs` has `isFile` and `isDirectory` for this purpose.

Reply:
`fs` doesn't easily have `isFile` https://nodejs.org/api/fs.html

I went with `existsSync` instead